server/data/data-library/src/main/java/org/apache/james/domainlist/lib/AbstractDomainList.java

```diff
@@ -118,6 +118,9 @@ public String getDefaultDomain() throws DomainListException {
         List<String> domains = getDomainListInternal();
```

Comment:
I tend to don't like variable re-affectation....

Here I would prefer a variable called "mutableDomains" to be created rather than seing domains reused.

By the way, nice spot.